docs/azure/aksextensions.md

```diff
@@ -0,0 +1,129 @@
+---
+Order: 7
+Area: azure
+TOCTitle: AKS Extensions
+ContentId: xxx
```

Comment:
💡 How do we get this `ContentId` generated? Please let me know. 🙏☕️

Reply:
You just need to generate a new GUID. I use the Create Guid extension (nwallace.createguid) but you can create a GUID anyway you want

Reply:
Awesome, and thank you so much @gregvanl , done, added, so its like any guid. ❤️☕️🥷 also Thanks @peterbom !!

Reply:
Yep, it's just so we can uniquely track a page in telemetry even if it is moved to another location or renamed

Reply:
Sounds great! Thanks heaps @gregvanl , do you know how can we get this inline to merge, I will leave that on your hand. this is first time I am contributing to this repo so not sure.

Reply:
@Tatsinnit The topic looks really good and I just left a few comments. Once those are addressed, I can merge sometime tomorrow and I can then give you a link to the staging server to give it a look over before we publish.

Reply:
Thank you so much @gregvanl ❤️🙏 sounds great, I will add your recommended changes soon.